src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js

```diff
@@ -738,7 +861,12 @@ describe('ContentUpdatesForm helpers', () => {
       });
 
       it('returns the correct object for the default case', () => {
-        expect(getDefaultActions('unknown', [], formatMessage))
+        expect(getDefaultActions({
+          option: 'unknown',
```

Comment:
Should we create constant for `unknown` ?

Reply:
Thanks for comment, not sure that it's required for this case, I'm using it only once in this test file